### PR TITLE
Amount editable enhancement

### LINKED ISSUE #93: Amount input: DEL as backspace icon, clear button
Screenshot 2023-05-11 at 2 18 57 PM

### PR DESCRIPTION
**Pull request by benalleng**
Closes #93

adds aditional functionality of amountEditable including fixed amounts add, holding backspace clears, and decimals for fiat are more accurate

### TODO
now that I've seen the figma there are some changes i need to make / revert back to

### CODE REVIEW

**Review by futurepaul — CHANGES_REQUESTED** [review]
sidechanneled with ben about getting this in line with the figma but it is a great improvement overall minus the "adding" functionality for the fixed amount buttons

**Comment by benalleng** [comment]
added new `font-inter` and changed the dialog window bg to match the figma as well as some other general changes to get the PR in line with the Figma

**Review by futurepaul — CHANGES_REQUESTED** [review]
sorry I should've mentioned: don't want inter. We use system font so it looks native-ish on ios and android.

File: src/styles/dialogs.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
 export const DIALOG_POSITIONER = "fixed inset-0 h-[100dvh] z-50";
 export const DIALOG_CONTENT =
-    "h-[100dvh] flex flex-col justify-between px-4 pt-4 pb-8 bg-neutral-800/80 backdrop-blur-xl touch-manipulation select-none";
+    "h-[100dvh] flex flex-col justify-between px-4 pt-4 pb-8 bg-[#171717] backdrop-blur-xl touch-manipulation select-none";
```
Comment (futurepaul) [inline src/styles/dialogs.ts:3]: prefer `bg-m-grey-800` (should already be a var in the tailwind)

Comment (futurepaul) [reply src/styles/dialogs.ts:3]: oh and don't need the backdrop blur anymore since there's no transparency

File: tailwind.config.cjs [hunk 2]
```diff
@@ -28,7 +28,8 @@ module.exports = {
                 "m-blue-dark": "hsla(220, 59%, 42%, 1)",
                 "m-red": "hsla(343, 92%, 54%, 1)",
                 "m-red-dark": "hsla(343, 92%, 44%, 1)",
-                "sidebar-gray": "hsla(222, 15%, 7%, 1)"
+                "sidebar-gray": "hsla(222, 15%, 7%, 1)",
+                "dialog-black": "rgba(23, 23, 23, 1)"
```
Comment (futurepaul) [inline tailwind.config.cjs:32]: the figma should show you which color var to use:

Screenshot 2023-06-26 at 5 28 02 PM

Comment (futurepaul) [reply tailwind.config.cjs:32]: so grey/800 = m-grey-800

Comment (benalleng) [reply tailwind.config.cjs:32]: sorry, didnt see your comment above this one making the change

Comment (futurepaul) [reply tailwind.config.cjs:32]: ah you need to rebase to master to see the new color vars in there

**Review by futurepaul — APPROVED** [review]
thank you!!

**Pull request merged**